glusterd2/snapshot/lvm/lvm.go

```diff
@@ -10,19 +10,20 @@ import (
 	"github.com/gluster/glusterd2/glusterd2/brick"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/pkg/api"
+
 	"github.com/pborman/uuid"
 )
 
 const (
-	//CreateCommand is path to lvm create
+	//CreateCommand is path to lvcreate
```

Comment:
For these binaries use the `exec.LookPath` instead of hardcoding their paths. It may not be the same everywhere. You may need to change these to be variables rather than constants for that to work.

Reply:
done